packages/o365/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.22.2"
```

```suggestion
- version: "2.23.0"
```
since this is an enhancement.